docs/faq.md

```diff
@@ -51,6 +51,13 @@ Development scalability was one of the main goals right from the beginning. Ther
 
 It is crucial that all micro frontends in a solution follow the same design guidelines. Luigi's default UI styling is based on [Fundamentals](https://sap.github.io/fundamental-styles/) but it can be customised. If you don’t want to use Fundamentals, but Bootstrap, Material, or something else instead, you need to re-style the Luigi view components according to your design guidelines or replace them with your own components completely.
 
+### I don't want to use the default Fiori Fundamentals style. How can I style Luigi differently?
+
+There are a few options to do that at the moment:
+- Use the Fundamental Styles theming capabilities which already allow you to achieve a lot by customizing the CSS variables. Find more info [here](https://github.com/SAP/theming-base-content).
```

Comment:
I'd add CSS/LESS/SASS variables instead of CSS only. All those variables like colors, font sizes, padding, etc. are part of SAP Fiori 3, not Fundamental Styles. It was extracted after version 0.8.1. I'm not sure if we need to. mention it to not confuse the client but wanted to clarify here at least for us.